input/kafkamdm/kafkamdm.go

```diff
@@ -307,3 +311,10 @@ func (k *KafkaMdm) Stop() {
 	k.client.Close()
 	offsetMgr.Close()
 }
+
+func (k *KafkaMdm) setClusterPrio() {
+	ticker := time.NewTicker(time.Second * 10)
+	for range ticker.C {
+		cluster.Manager.SetPriority(k.lagMonitor.Metric())
+	}
+}
```

Comment:
it would be cleaner if we were able to exit out of this goroutine when the plugin stops. in this case it doesn't matter much since we only call Stop() when stopping MT, but that may change.